docs/setup/docker/getting-started-docker.md

```diff
@@ -0,0 +1,95 @@
+# Getting Started with Nuclio on Docker
+
+Follow this step-by-step guide to set up a Nuclio development environment that uses Docker
+
+#### In this document
+
+- [Prerequisites](#prerequisites)
+- [Run Nuclio](#run-nuclio)
+- [Deploy a function with the Nuclio dashboard](#deploy-a-function-with-the-nuclio-dashboard)
+- [Deploy a function with the Nuclio CLI (nuctl)](#deploy-a-function-with-the-nuclio-cli-nuctl)
```

Comment:
Suggest using present progressive for titles

Reply:
I understand there's a consistency issue with other docs.
Will do this in a separate PR (or not) after consulting with @Sharon-iguazio